Apps/Sandcastle/gallery/Terrain.html

```diff
@@ -136,6 +137,7 @@
 var terrainSamplePositions;
 
 function sampleTerrainSuccess() {
+    viewer.scene.globe.depthTestAgainstTerrain = true;
```

Comment:
Shouldn't we just set this once in the beginning immediately after Viewer construction?  I would also add a comment explaining what it is and why we need it because it's a common question on the mailing list.